### DISCUSSION TITLE
How to run this project in web?

### DISCUSSION THREAD

**Discussion by miroshar-success** [implerhq/impler.io#585] [Q&A]
Is there anyone to help me to run this project in web?

**Comment by chavda-bhavik** [comment]
@OskarMast Sorry for delay. How is your status with setting up Impler? Do you need any help?

**Reply by chavda-bhavik** [reply]
@OskarMast Can you try the latest [contributing guide](https://github.com/implerhq/impler.io/blob/next/CONTRIBUTING.md)?
